test/test.js

```diff
@@ -776,9 +776,32 @@ describe('vend-nodejs-sdk', function () {
       });
     });
 
-    describe('with inventory API', function() {
+    describe.only('with versions API', function () {
+      this.timeout(300000);
+
+      it('can fetch latest versions of all data in Vend', function () {
+        return vendSdk.versions.fetchAll(getConnectionInfo())
+          .then(function (response) {
+            expect(response).to.exist;
+            expect(response.data).to.exist;
+            expect(response.data).to.be.instanceof(Object);
```

Comment:
@kamal0808 lets also check for some known common keys in the payload ... It will help alert us when the test breaks due to changes from their side.